### PR TITLE
add tests

### PR DESCRIPTION
**Pull request by alexeyMohnatkin**
This change is [Reviewable](https://reviewable.io/reviews/shakacode/sass-resources-loader/52)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,6 +46,10 @@
     "babel-preset-es2015": "^6.1.18",
     "eslint": "^1.10.1",
     "eslint-config-airbnb": "^1.0.0",
```
Comment (alexfedoseev) [inline package.json]: You should remove `eslint-config-airbnb` if you installed `eslint-config-shakacode`

File: test/scss/imports.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+div {
+    display: block;
+    color: $text-color;
+}
```
Comment (alexfedoseev) [inline test/scss/imports.scss]: A new line (configure editor).

File: .gitignore [hunk 3]
```diff
@@ -3,3 +3,5 @@ node_modules/
 *.log
 npm-debug.log*
 .DS_Store
+test/output
+.vscode
```
Comment (alexfedoseev) [inline .gitignore]: A new line (configure editor).

File: .eslintignore [hunk 4]
```diff
@@ -1,3 +1,4 @@
 node_modules/
 lib/
 example/
+test/output
```
Comment (alexfedoseev) [inline .eslintignore]: A new line (configure editor).

Comment (alexeyMohnatkin) [reply .eslintignore]: I've added .editorconfig instead

File: package.json (hunk 1 above)
Comment (alexeyMohnatkin) [reply package.json]: Actually it was you)
Without this linting fails

**Pull request merged**